### ISSUE TITLE
missing cytoscape demo json file

### ISSUE THREAD

**Issue by l0o0** [christabor/flask_jsondash#145]
When I was playing around with flask_jsondash, I found I could not display figure in cytoscape. 
It seems degre.json file is missing in example_app/examples/cytoscape folder in `request.args.get('name', 'dagre')`.  
Did I forget to set some values to chart?

**Comment by christabor** [comment]
Sorry I didn't respond sooner, been out as of recent! I will look at this asap.

**Comment by christabor** [comment]
Dagre should not be required for Cytoscape, they are separate. What is the error you are getting (in flask, and/or in your browser console)?

**Comment by christabor** [comment]
You are right, the default value is invalid. Looks like a copy paste error when setting up the demo endpoint. The options are available in this example: https://github.com/christabor/flask_jsondash/blob/master/example_app/examples/config/cytoscape.json, which should clear it up. I'm also adding a quick fix to that default arg.
